New version: GenomicAnnotations v0.3.7

JuliaRegistrator opened the pull request:
- Registering package: GenomicAnnotations
- Repository: https://github.com/BioJulia/GenomicAnnotations.jl
- Created by: @kdyrhage
- Version: v0.3.7
- Commit: 67f82c444f3c8cbcb7cf68ec27f81df46ca08b75
- Reviewed by: @kdyrhage
- Reference: https://github.com/BioJulia/GenomicAnnotations.jl/commit/67f82c444f3c8cbcb7cf68ec27f81df46ca08b75#commitcomment-139625326
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd62da6390bdefbbb815843a5f0745260cca59d545627f174823d86f12df4244ab27fd1c932723ca1bd8e2bddfb4a874fd373310188f86061895e9b220cf5114e94ad7bfd882ecc26b7689cfa45fa72f74e38a4a4a889e9a0b20070cbe6b64173ee0a11eb23344d73d987630ed4fcdaba32fad62493e986c95b1d1300283b53444d773d0537971f02858b85dfc56bedc35d72b40482119e4eb504c196312fe3bd3ed8e620482bded1b47720832f8708aa0b -->

DilumAluthge commented:
[noblock]

Looks like there is a merge conflict.

@kdyrhage Can you re-trigger Registrator? This will fix the merge conflict.

JuliaRegistrator committed:
New version: GenomicAnnotations v0.3.7

UUID: 4f8a0a0a-376d-5ac0-ab14-e88793df67f0
Repo: https://github.com/BioJulia/GenomicAnnotations.jl.git
Tree: bf2ba100bcb900238794df5e8bd61602324d1866

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
